test/metabase/query_processor/middleware/optimize_temporal_filters_test.clj

```diff
@@ -10,21 +10,26 @@
    [metabase.test :as mt]
    [metabase.util.date-2 :as u.date]))
 
-(driver/register! ::timezone-driver, :abstract? true)
+(deftest ^:parallel temporal-arithmetic?-test
+  (is (#'optimize-temporal-filters/temporal-arithmetic?
+       [:+ [:field 1 {:base-type :type/DateTime}] [:interval -2 :month]])))
```

Comment:
A couple of negative tests wouldn't be a bad idea.

Reply:
ended up removing this stuff